nbviewer/providers/gist/handlers.py

```diff
@@ -50,12 +59,18 @@ class UserGistsHandler(GistClientMixin, BaseHandler):
     .ipynb file extension is required for listing (not for rendering).
     """
     def render_usergists_template(self, entries, user, provider_url, prev_url,
-                                 next_url, **namespace):
+                                  next_url, **namespace):
+        """
+        provider_url: str
+            URL to the notebook document upstream at the provider (e.g., GitHub)
+        executor_url: str, optional
```

Comment:
Is `executor_url` passed in `namespace`? If so, we should explicitly state that.